### PR TITLE
Support promises.

### PR DESCRIPTION
**Pull request by dlongley**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
   "homepage": "https://github.com/digitalbazaar/bedrock-redis",
   "dependencies": {
     "backo": "^1.1.0",
+    "promise-redis": "0.0.5",
     "redis": "^2.8.0"
```
Comment (mattcollier) [inline package.json:27]: looks like we no longer need this dep?

**Review by mattcollier — CHANGES_REQUESTED** [review]
That looks nice.

File: package.json (hunk 1 above)
Comment (davidlehn) [reply package.json:27]: Still need the redis dep. promise-redis just depends on "*" so want to restrict it to what we support.

Comment (dlongley) [reply package.json:27]: @davidlehn is correct -- and really `promise-redis` should be using peerDeps for that, but whatever.

Comment (mattcollier) [reply package.json:27]: I think it would be a good idea to put that information in the changelog.

**Review by mattcollier — APPROVED** [review]

**Pull request merged**